parser/src/ast/mod.rs

```diff
@@ -48,6 +48,7 @@ pub enum Expr {
     Next(Identifier),
     Add(Box<Expr>, Box<Expr>),
     Subtract(Box<Expr>, Box<Expr>),
+    Exp(Box<Expr>, Box<Expr>),
```

Comment:
this is super nitty, but since all of the other variants are full words, I think maybe we should use `Exponent` instead of `Exp`. Either that, or we could switch to `Sub`, `Mul`, `Const`, `Var`, which would be fine too (and maybe better?) Feel free to pick whichever you prefer, as long as we're consistent. (In fact, in the graph in the IR I probably should have used `Const`)

Reply:
I agree keeping the concise version seems better. So changed these to `Sub`, `Mul`, `Const`, `Var`, etc.